presentation/src/test/java/com/android254/presentation/sessions/mappers/TimePeriodTest.kt

```diff
@@ -56,8 +56,8 @@ class TimePeriodTest(
         )
         assertEquals(
             "Should be valid period",
-            periodResult,
-            formattedTime.period
+            periodResult.lowercase(),
+            formattedTime.period.lowercase()
```

Comment:
why is it necessary to change the case?

Reply:
<img width="694" alt="Screenshot 2023-02-24 at 12 20 03 PM" src="https://user-images.githubusercontent.com/40160345/221140564-d21f7bc3-af4d-486b-bd9b-784340692a17.png">
Was testing it out and found out it's different in different locations. Here's a link to a plaground for further tests :
https://pl.kotl.in/l0xT-QI4p

Reply:
@chepsi is this resolved